sections/futures/Trade/NextPriceConfirmationModal.tsx

```diff
@@ -80,7 +80,7 @@ const NextPriceConfirmationModal: FC<NextPriceConfirmationModalProps> = ({
 	const orderDetails = useMemo(() => {
 		const newSize = side === PositionSide.LONG ? tradeSize : -tradeSize;
 
-		return { size: (positionSize ?? zeroBN).add(newSize) };
+		return { size: (positionSize ?? zeroBN).add(newSize).abs() };
```

Comment:
Won't this still be incorrect if someone is short and they are increasing their short? I think we need to adjust `positionSize` based on their position side as well.

Reply:
@Tburm I think position size is already either positive/negative depending on what side the position is on. If I recall correctly, that's actually how we get the side of the position on the frontend.